projects/swc-starter-webpack/src/extended_swc_samples/my-action-bar.js

```diff
@@ -18,7 +18,10 @@ class MyActionBar extends ActionBar {
         ActionBar.styles,
         css`
             :host([open]) {
-                background-color: antiquewhite;
+                background-color: darkgray;
+                padding: 10px;
+                border-radius: 10px;
+                box-shadow: 0px 0px 30px darkgrey;
```

Comment:
Why are we adding border-radius and box-shadow for extended sample?

Reply:
This is just to create my custom `my-action-bar` extended from `sp-action-bar` where I am trying to create an extra box-shadow around the` sp-action-bar` to make `my-action-bar` look different from the `sp-action-bar`.